lib/absinthe/phase/document/arguments/normalize.ex

```diff
@@ -17,18 +17,30 @@ defmodule Absinthe.Phase.Document.Arguments.Normalize do
 
   @spec run(Blueprint.t, Keyword.t) :: {:ok, Blueprint.t}
   def run(input, _options \\ []) do
-    acc = %{provided_values: %{}}
+    acc = %{provided_values: fetch_provided_values(input)}
```

Comment:
Unfortunately this suggestion didn't work as it caused the existing `normalize_test` cases to fail. This is because the blueprint has >1 operations and no provided `operation_name` needed to mark one of the operations as `current=true`. This did expose a flaw in my original implementation that was only collecting `provided_values` from the first operation, so I've updated my solution and the test to reflect that.